docs/bom/generate.md

```diff
@@ -0,0 +1,47 @@
+# bom generate
+
+Create SPDX compliant Bill of Materials 
+
+- [Summary](#summary)
+- [Installation](#installation)
+- [Usage](#usage)
+
+## Summary 
+
+`bom generate` is the subcommand to generate SPDX manifests.
+Currently supports creating SBOM for files, images, and docker
+archives (images in tarballs). Supports pulling images from
+registries.
+
+bom can take a deeper look into images using a growing number
```

Comment:
```suggestion
`bom` can take a deeper look into images using a growing number
```

Reply:
I've fixed this one. It came out like that because this is an output of the actual command help (like the docs in krel).